kaitaistruct.py

```diff
@@ -15,51 +20,171 @@
 __version__ = '0.9'
 
 
-class KaitaiStruct(object):
-    def __init__(self, stream):
-        self._io = stream
+class _KaitaiStruct_:
```

Comment:
Do you have any ideas how to better name it? Underscores are usual convention for marking something as `use with care`, so `_KaitaiStruct_` is double-`use with care`, but I shouldn't prepend a double underscore because it is reserved in python.

Reply:
Underscores are the usual convention for internal things that you're not really supposed to touch as an external user, so it's not a good naming convention for classes that users are meant to subclass...

Why not `KaitaiStructBase` for the base class, and something like `NonClosingKaitaiStruct` for the other class?